internal/model/id.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import (
+	"bytes"
+	"encoding/base32"
+
+	"github.com/pborman/uuid"
+)
+
+var encoding = base32.NewEncoding("ybndrfg8ejkmcpqxot1uwisza345h769")
+
+// NewID is a globally unique identifier.  It is a [A-Z0-9] string 26
+// characters long.  It is a UUID version 4 Guid that is zbased32 encoded
+// with the padding stripped off.
+func NewID() string {
```

Comment:
Why not use `model.NewId()` for this? https://github.com/mattermost/mattermost-server/blob/master/model/utils.go#L140